lib/api/request/kuzzleRequest.ts

```diff
@@ -849,13 +849,9 @@ export class KuzzleRequest {
     errorName: string,
     def: number | null = null
   ): number {
-    let value = get(obj, name);
+    let value = get(obj, name, def) || def;
```

Comment:
Why do we need to add the `|| def` part? Lodash does not returns the default param directly?